### DISCUSSION TITLE
Help on learning netty 4

### DISCUSSION THREAD

**Discussion by softwareDeveloper2014** [community/community#87318] [Programming Help]
### Body

Hi,
please give some tutorials on learning netty 4

Please only recommend tutorials/video courses or other stuff, you are used successfully.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by anuragjain-git** [comment]
CSS typically involves demonstrating understanding of front-end development and the ability to create responsive and visually appealing user interfaces. TypeScript is not strictly necessary for all junior roles though, can be a valuable skill to have.

TypeScript
1. [Youtube](https://www.youtube.com/watch?v=zeCDuo74uzA)
2. [Docs: typescriptlang](https://www.typescriptlang.org/docs/)

CSS Resources  (Docs):
1. [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS)
2. [W3School](https://www.w3schools.com/w3css/)
3. [cssreference](https://cssreference.io/)

Youtube Videos:
1. [FreeCodeCamp](https://www.youtube.com/watch?v=OXGznpKZ_sA)
2. [SuperSimpleDev](https://www.youtube.com/watch?v=G3e-cpL7ofc)

**Comment by jonasermert** [comment]
[Awesome Javascript](https://github.com/sorrycc/awesome-javascript)
[Awesome CSS](https://github.com/awesome-css-group/awesome-css)

https://github.com/panaverse/learn-typescript

**Comment by ghost** [comment]
Hi @s2023alek I guess these would help...

## Essentials of CSS to get hired as a junior

- Box-Model
- Positions
  - Absolute
  - Relative
  - Static
  - Fixed
  - Sticky
- Layouts
  - Flexbox
  - Grid
  - Float
- Units like px, vw, vh, %, rem, em,etc
- Features like calc(), var(), min(), max()
- Responsiveness - Media queries
- Animation - keyframes
- Transition
- SAAS (optional)
- Dev Tools

## Tutorial I Would Suggest

### Tutorials from Davegray
- [CSS](https://youtu.be/OXGznpKZ_sA?si=3tcfypSyBK0ci-9A)
- [Typescript](https://youtu.be/gieEQFIfgYc?si=jSNysg5Iu2W7bAyK)

### Online Resources
- [w3schools](https://www.w3schools.com/)
- [MDN Docs](https://developer.mozilla.org/en-US/)
